### ISSUE TITLE
Faster adding a watches

### ISSUE THREAD

**Issue by kjaft** [awulkiew/graphical-debugging#17]
Hi, I have another Idea :) 
It would be quite handy if one could add entries to one of the graphical wathces directly by context menu from the locals/autos/watches window selection. Or even directly from the source code context menu (like you can choose "Add Watch/Add Quick Watch...: there). I am not sure if this is technically possible but to me it would be a huge workflow ease.

**Comment by awulkiew** [comment]
Yes, these are also good ideas. It has to be technically possible because things like this are done in VS. It'd require some digging to know how to do it though.

Btw, if you'd like to play with it feel free to clone the repo and propose a PR. :)
